applications/GeoMechanicsApplication/tests/test_set_multiple_moving_load_process.py

```diff
@@ -1,12 +1,54 @@
 import KratosMultiphysics
-
 import KratosMultiphysics.GeoMechanicsApplication as GMA
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
-class TestSetMultipleMovingLoadsProcess(KratosUnittest.TestCase):
+
+class KratosGeoMechanicsSetMultipleMovingLoadProcessTests(KratosUnittest.TestCase):
     time_dependent_load = '["0.0", "-2.0*t", "0.0"]'
     time_dependent_velocity = ' "2.0*t" '
 
+    def set_nodal_displacement_dof(self):
+        for node in self.mp.Nodes:
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_X)
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_Y)
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_Z)
+
+    @staticmethod
+    def setup_strategy(mp):
+        """
+        Setup default strategy for solving the problem
+
+        Parameters
+        ----------
+        mp: model part
+
+        Returns default strategy
+        -------
+
+        """
+
+        linear_solver = KratosMultiphysics.SkylineLUFactorizationSolver()
+        builder_and_solver = KratosMultiphysics.ResidualBasedBlockBuilderAndSolver(linear_solver)
+        scheme = KratosMultiphysics.ResidualBasedIncrementalUpdateStaticScheme()
+        convergence_criterion = KratosMultiphysics.ResidualCriteria(1e-1, 1e-1)
+        convergence_criterion.SetEchoLevel(0)
+
+        max_iters = 10
+        compute_reactions = False
+        reform_step_dofs = False
+        move_mesh_flag = False
+        strategy = KratosMultiphysics.ResidualBasedNewtonRaphsonStrategy(mp,
+                                                                         scheme,
+                                                                         convergence_criterion,
+                                                                         builder_and_solver,
+                                                                         max_iters,
+                                                                         compute_reactions,
+                                                                         reform_step_dofs,
+                                                                         move_mesh_flag)
+
+        strategy.SetEchoLevel(0)
+        return strategy
+
     def setUp(self):
         self.current_model = KratosMultiphysics.Model()
         self.rmp = self.current_model.CreateModelPart("root_part")
```

Comment:
Since we're improving these tests a lot anyway, I would propose to rename these members: rmp -> root_model_part, mp -> model_part etc.